QUICKSTART.md

```diff
@@ -271,6 +271,32 @@ BoundLongCounter someWorkCounter = counter.bind("Key", "SomeWork");
 // Record data
 someWorkCounter.add(123);
 
+// Alternatively, the user can use the unbounded counter and explicitly
+// specify the labels set at call-time:
+counter.add(123, "Key", "SomeWork");
+```
+
+`Observer` is an additional instrument supporting an asynchronous API and
+collecting metric data on demand, once per collection interval.
+
+The following is an example of observer usage:
+
+```java
+// Build observer e.g. LongObserver
+LongObserver observer = meter
+        .observerLongBuilder("doWork_latency")
+        .setDescription("gRPC Latency")
```

Comment:
Latency is not a good example for this. Memory Usage/ Cpu Usage is better.

Reply:
Fixed. Btw, I took this from the inline docs from `LongObserver.java` so we should do a follow up to update those ones as well.